### PR TITLE
Demo

### PR DESCRIPTION
**Pull request by billyxu008**
Added code that use react for frontend and python for backend services.

### CODE REVIEW

File: CNAME
Comment (avanegas95) [inline CNAME:1]: I had added this in but we can delete it. We only need this for GitHub pages. AWS domain hosting uses their custom DNS

File: README.md
Comment (avanegas95) [inline README.md:1]: Should we specify running the backend as well locally?

File: public/logo512.png
Comment (avanegas95) [inline public/logo512.png:1]: Didn't realize you made logos already! Nice!

**Review by avanegas95 — APPROVED** [review]
lgtm

File: CNAME
Comment (billyxu008) [reply CNAME:1]: Well, I think we don't need to worry about hosting it on Github for now.

File: README.md
Comment (billyxu008) [reply README.md:1]: Yeah, I can add some documentation first if you want.

File: CNAME
Comment (avanegas95) [reply CNAME:1]: Removed

**Pull request merged**